README.md

```diff
@@ -425,6 +425,7 @@ should be conducted in.
 | `cooldown_seconds`                     | `0`              | The number of seconds before the end of the test to exclude in reporting (only used for report generation)                                  |
 | `license_key`                          | `your_ee_key`    | The Hazelcast Enterprise Edition license to use in your test, if using `hazelcast-enterprise5` drivers                                      |
 | `parallel`                             | `True`           | Defines whether tests should be run in parallel when multiple tests are defined within 1 suite (default false)                              |
+| `cp_priorities` | <pre>- address: internalIp<br> &nbsp;priority: 1</pre> | Defines the leadership priority of the CP Subsystem members in the cluster. Use the internal IP address of the agent you wish to configure. |
```

Comment:
It would be nice to have more details for this functionality, it's not as simple as the others in this table - perhaps adding a small section to the end of the README with an example config to clearly demonstrate correct usage?

Reply:
done [fd63f6c](https://github.com/hazelcast/hazelcast-simulator/pull/2222/commits/fd63f6c3785df759fe726cbecb261a87b3562209)